README.md

```diff
@@ -170,51 +182,47 @@ A number of callback functions are fired at various points of the flow. The most
     containerId: 'onfido-mount',
     // here we send the data in the complete callback
     onComplete: function() {
-      var captures = Onfido.getCaptures()
-      sendToServer(captures)
+      //communicate with your backend and initiate the check
     }
   })
 
-  const reduceObj = (object, callback, initialValue) =>
-    Object.keys(object).reduce(
-      (accumulator, key) => callback(accumulator, object[key], key, object),
-      initialValue)
-
-  const objectToFormData = (object) =>
-    reduceObj(object, (formData, value, key) => {
-      formData.append(key, value)
-      return formData;
-    }, new FormData())
+  ```
+  The `getCaptures` function contains the document and face files captured during the flow. Based on the applicant id, you can then create a check for the user via your backend. Here is an example of how to initiate a check from your backend.
 
-  const postData = (data, endpoint, callback) => {
-    const request = new XMLHttpRequest()
-    request.open('POST', endpoint, true)
+  ```js
```

Comment:
I don't think this example is necessary. The fact it is in JS also is confusing as it suggests this could be run on the client. I think we should just link to https://documentation.onfido.com/#create-check here